docs/user/visualize.asciidoc

```diff
@@ -132,8 +132,11 @@ include::{kib-repo-dir}/visualize/lens.asciidoc[]
 include::{kib-repo-dir}/visualize/most-frequent.asciidoc[]
 
 include::{kib-repo-dir}/visualize/tsvb.asciidoc[]
+
 include::{kib-repo-dir}/visualize/timelion.asciidoc[]
 
+include::{kib-repo-dir}/visualize/timelion-deprecation.asciidoc[]
```

Comment:
I would relocate this file, and embed it at the end of the Timelion content. This is probably where users will find it the best.